ksql-engine/src/main/java/io/confluent/ksql/planner/plan/KsqlStructuredDataOutputNode.java

```diff
@@ -273,6 +332,25 @@ public KsqlStructuredDataOutputNode cloneWithDoCreateInto(final boolean newDoCre
     );
   }
 
+  private static class SourceTopicProperties {
+
+    private final int partitions;
+    private final short replicas;
+
+    SourceTopicProperties(final int partitions, final short replicas) {
+      this.partitions = partitions;
+      this.replicas = replicas;
+    }
+
+    public int getPartitions() {
```

Comment:
You can remove the access methods and just access the private fields directly when its a nested class = less code.